instrumentation/sinatra/lib/opentelemetry/instrumentation/sinatra/middlewares/tracer_middleware.rb

```diff
@@ -46,6 +44,7 @@ def trace_response(span, env, resp)
             span.set_attribute('http.status_code', status)
             span.set_attribute('http.status_text', ::Rack::Utils::HTTP_STATUS_CODES[status])
             span.set_attribute('http.route', env['sinatra.route'].split.last) if env['sinatra.route']
+            span.name(env['sinatra.route'].split.last) if env['sinatra.route']
```

Comment:
```suggestion
            span.name = env['sinatra.route'].split.last if env['sinatra.route']
```

Reply:
`name` looks to me to be a method just like `set_attribute` so I'd think the syntax is the same... am I missing something?

Reply:
It's a "setter" (aka `attr_reader`), not a normal method: https://github.com/open-telemetry/opentelemetry-ruby/blob/bb1ad6be6f56e39b9c26e69febbc87ef19500c34/api/lib/opentelemetry/trace/span.rb#L114

Reply:
Status is a similar thing: https://github.com/open-telemetry/opentelemetry-ruby/blob/bb1ad6be6f56e39b9c26e69febbc87ef19500c34/api/lib/opentelemetry/trace/span.rb#L103

Reply:
Ahhh, thank you for educating me (seriously)